app/lib/actions/candlepin/candlepin_listening_service.rb

```diff
@@ -59,7 +59,7 @@ def start(suspended_action)
         else
           suspended_action.notify_not_connected("Not Connected")
         end
-      rescue TransportFailure => e
+      rescue => e
```

Comment:
@ehelms of course its that simple :) that seems to work just fine, updated the commit.